code/modules/surgery/surgery_step.dm

```diff
@@ -116,8 +124,15 @@
 			if(failure(user, target, target_zone, tool, surgery, fail_prob))
 				play_failure_sound(user, target, target_zone, tool, surgery)
 				advance = TRUE
-			if(chem_check_result)
-				return .(user, target, target_zone, tool, surgery, try_to_fail) //automatically re-attempt if failed for reason other than lack of required chemical
+		if(!HAS_TRAIT(target, TRAIT_SURGERY_PREPARED) && target.stat != DEAD && !IS_IN_STASIS(target) && fuckup_damage) //not under the effects of anaesthetics or a strong painkiller
+			if(!HAS_TRAIT(user, TRAIT_SURGEON)) // The patient needs more mouse bites
```

Comment:
abductors are currently the only thing that gets this, it also lets them do ANY surgery i think.